Release/src/http/client/http_client_winhttp.cpp

```diff
@@ -462,7 +465,7 @@ class winhttp_client : public _http_client_communicator
         if(WINHTTP_INVALID_STATUS_CALLBACK == WinHttpSetStatusCallback(
             m_hSession,
             &winhttp_client::completion_callback,
-            WINHTTP_CALLBACK_FLAG_ALL_COMPLETIONS | WINHTTP_CALLBACK_FLAG_HANDLES,
+			WINHTTP_CALLBACK_FLAG_ALL_COMPLETIONS | WINHTTP_CALLBACK_FLAG_HANDLES |	WINHTTP_CALLBACK_FLAG_SEND_REQUEST,
```

Comment:
I guess it should be WINHTTP_CALLBACK_STATUS_SENDING_REQUEST instead of WINHTTP_CALLBACK_FLAG_SEND_REQUEST.